conf/machine/include/qcom-base.inc

```diff
@@ -3,8 +3,8 @@
 # Provider for linux kernel
 PREFERRED_PROVIDER_virtual/kernel ?= "linux-yocto-dev"
 
-KERNEL_IMAGETYPE ?= "Image"
-KERNEL_IMAGETYPES ?= "Image.gz"
+KERNEL_IMAGETYPE ?= "Image.gz"
+KERNEL_IMAGETYPES ?= "Image"
```

Comment:
I believe that these 2 variables are 'merged' together in OE. as such I don't know how this change makes any difference. Can you please point out to the problem you have that led to this patch?

Reply:
The issue is that https://github.com/Linaro/meta-qcom/blob/master/recipes-kernel/images/linux-qcom-uki.bb#L117 is not consuming the variable after the merge (which is done in the kernel class), so either we invert as done here or we change meta-qcom to drop take the variable merge into account.

Reply:
thanks for the context. We need to fix meta-qcom instead. something like it's done in meta/classes-recipe/linux-dummy.bbclass. These variables are expected to be used in the kernel recipe, so using them outside of the kernel recipe is 1) not expected, 2) potentially the wrong thing to do. 

Perhaps we need to review why linux-qcom-uki.bb was done like that.